src/Font.cpp

```diff
@@ -507,7 +507,7 @@ void Font::LoadFontPageSettings( FontPageSettings &cfg, IniFile &ini, const RStr
 
 				wchar_t c;
 				if( sCodepoint.substr(0, 2) == "U+" && IsHexVal(sCodepoint.substr(2)) )
-					sscanf( sCodepoint.substr(2).c_str(), "%x", &c );
+					sscanf( sCodepoint.substr(2).c_str(), "%x", (unsigned int *)&c );
```

Comment:
As a heads up, I prefer c++ style casts. See if using `static_cast<unsigned int *>(&c)` will help. Or maybe even better, just `static_cast<unsigned int>(c)`. I'm not seeing why a pointer is needed here.